### PR TITLE
Add BZ automation to repo

### PR DESCRIPTION
**Pull request by rayfordj**
Adding the Bugzilla Automation to your Repo (from [bz-github-action](https://github.com/konveyor/bz-github-action#adding-the-bugzilla-automation-to-your-repo))

### CODE REVIEW

File: .github/workflows/pr-merge.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      - uses: docker://quay.io/konveyor/pr-merge-github-action
+        name: update bugzilla to modified
+        with:
+          bugzilla_token: ${{ secrets.BUGZILLA_TOKEN }}
+          org_repo: ${{ github.repository }}
+          pr_number: ${{ github.event.pull_request.number }}
+          bz_product: "Migration Toolkit for Containers"
+          title: ${{ github.event.pull_request.title }}
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          branch_to_release: "release-1.4.0:1.4.0,master:1.5.0"
```
Comment (shawn-hurley) [inline .github/workflows/pr-merge.yml:36]: I see a release-1.3.2 branch but it is not in the mapping

**Review by shawn-hurley — COMMENTED** [review]
LGTM except for the one question

File: .github/workflows/pr-merge.yml (hunk 1 above)
Comment (rayfordj) [reply .github/workflows/pr-merge.yml:36]: I removed it because we never actually had a 1.3.2 for this repo.

**Review by djwhatle — APPROVED** [review]
As long as Shawn has taken and look and ACKs, LGTM. Thanks @rayfordj !

**Pull request merged**